### PR TITLE
[Fix] Fix unit test in dev1.x

### PR DESCRIPTION
**Pull request by grimoire**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily receiving feedbacks. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

MMCV has update the api of nms rotated, which broken our rewriter.
https://github.com/open-mmlab/mmcv/blob/2.x/mmcv/ops/nms.py
## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit tests to ensure the correctness.
3. If the modification has a dependency on downstream projects of a newer version, this PR should be tested with all supported versions of downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by grimoire** [comment]
> The UT still failed in CI, maybe need to check.

Errr, you are right.
The error comes with an update in mmcv nms_rotated ops. Should we update the mmcv version used in ci @lvhan028 ?

**Comment by lvhan028** [comment]
UT still failed.
@AllentDan regarding the lint issue, should we update config or links in doc?

**Comment by AllentDan** [comment]
Yes. @grimoire Could you please help me replace all `https://github.com/open-mmlab/mmdeploy/blob/1.x/configs/mmdet/detection/detection_rknn_static-320x320.py` with `https://github.com/open-mmlab/mmdeploy/blob/1.x/configs/mmdet/detection/detection_rknn-int8_static-320x320.py` in this PR?

**Comment by grimoire** [comment]
@AllentDan Sure

File: docs/en/01-how-to-build/rockchip.md [hunk 1]
```diff
@@ -172,7 +172,7 @@ label: 65, score: 0.95
       ])
   ```
 
-  RetinaNet & SSD & FSAF with rknn-toolkit2, you may paste the following partition configuration into [detection_rknn_static-320x320.py](https://github.com/open-mmlab/mmdeploy/tree/1.x/configs/mmdet/detection/detection_rknn_static-320x320.py). Users with rknn-toolkit can directly use default config.
+  RetinaNet & SSD & FSAF with rknn-toolkit2, you may paste the following partition configuration into [detection_rknn_static-320x320.py](https://github.com/open-mmlab/mmdeploy/blob/dev-1.x/configs/mmdet/detection/detection_rknn-int8_static-320x320.py). Users with rknn-toolkit can directly use default config.
```
Comment (lvhan028) [inline docs/en/01-how-to-build/rockchip.md:175]: may use `https://github.com/open-mmlab/mmdeploy/blob/1.x/configs/mmdet/detection/detection_rknn-int8_static-320x320.py`, which helps when migrating 1.x to main

**Review by hanrui1sensetime — APPROVED** [review]
LGTM.

**Review by tpoisonooo — APPROVED** [review]

**Pull request merged**